aiida-registry-app/src/Components/MainIndex.jsx

```diff
@@ -235,6 +249,42 @@ export function MainIndex() {
           </Box>
           </div>
 
+          {isSearchSubmitted === true ? (
+            <>
+            <h2>Showing {searchResults.length} pages matching the search query.</h2>
+            {searchResults.length === 0 && (
+              <div>
+            <h3 className='submenu-entry' style={{textAlign:'center'}}>Can't find what you're looking for?<br/> Join the AiiDA community on Discourse and open a discussion here. <br/> <a href='https://aiida.discourse.group' target="_blank">https://aiida.discourse.group</a></h3>
+            </div>
+            )}
+                {searchResults.map((suggestion) => (
+                  <>
+                <div className='submenu-entry'>
+                    <Link to={`/${suggestion.item.name}`}><h3 key={suggestion.item.name} className="suggestion-item">
+                      {suggestion.item.name}
+                    </h3></Link>
+                    <ul>
+
+                    {suggestion.matches.map((match) => (
+                      <>
+                      {typeof match.key === 'object' && (
+                        <>
+                    <Link to={`/${suggestion.item.name}#${match.key[1]}.${match.key[2]}`}><li key={match.key} className="suggestion-item">
+                      {match.key[2]}
+                    </li></Link>
+                    <p>{extractSentenceAroundKeyword(match.value, searchQuery)} ...</p>
+                        </>
+                      )}
+                    </>
+                    ))}
+                </ul>
+                    </div>
+                    </>
+                  ))}
+            </>
+
+        ):(
```

Comment:
What is this? :( looks like a facial expression?

Reply:
Haha, it is the React syntax, not my current mood😅
It is a conditional rendering in React like if/else statements
```
{isSearchSubmitted === true ? ( 
//display search HTML
):(
//display all the plugins index
 ) }
```